File: 3689-maximum-total-subarray-value-i/3689-maximum-total-subarray-value-i.cpp
class Solution {
public:
    long long maxTotalValue(vector<int>& nums, int k) {
        long long mini=INT_MAX;
        long long maxi=INT_MIN;
        for(int i:nums){
            mini=min(mini,(long long)i);
            maxi=max(maxi,(long long)i);
        }
        return (maxi-mini)*(k);
    }
};

File: 3689-maximum-total-subarray-value-i/3689-maximum-total-subarray-value-i.java
class Solution {
    public long maxTotalValue(int[] nums, int k) {
        long min=Integer.MAX_VALUE;
        long max=Integer.MIN_VALUE;
        for(int i:nums){
            min=Math.min(min,i);
            max=Math.max(max,i);
        }
        return (max-min)*(k);
    }
}